cloudformation/media-atom-maker.json

```diff
@@ -488,7 +488,8 @@
                 "APP": "media-atom-uploader",
                 "STAGE": { "Ref": "Stage" },
                 "CONFIG_BUCKET": { "Ref": "ConfigBucket" },
-                "CONFIG_KEY": { "Fn::FindInMap": [ "LambdaConfig", "Uploader", { "Ref": "Stage" }] }
+                "CONFIG_KEY": { "Fn::FindInMap": [ "LambdaConfig", "Uploader", { "Ref": "Stage" }] },
+                "DOMAIN": { "Fn::Join": ["", [ "https://", { "Ref": "DomainToMonitor" } ]] }
```

Comment:
As discussed IRL, I will remove this and replace it with the `host` setting from `media-atom-maker.private.conf`